3rdWeek/GH/회의실 배정.js

```diff
@@ -0,0 +1,14 @@
+let input = require('fs').readFileSync('코테스터디/예제.txt').toString().trim().split('\n');
+input.shift().split(' ')
+let 회의리스트 = input.map((v) => v.split(' '))
```

Comment:
```suggestion
let 회의리스트 = input.map((v) => v.split(' ').map(Number))
```
와 같이 type을 `Number`로 바꿔주면 아래에서 `+변수`를 신경쓰지 않아도 되서 편할거같아요!

Reply:
맞네영,, 받을때 바로 처리 하는게 훨씬 깔끔하고 보기 쉽겠네용,, 피드백 감사합니다 ㅎ